packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -69,8 +91,14 @@ class OpPerfTelemetry {
         this.deltaManager.on("disconnect", () => {
             this.sequenceNumberForMsnTracking = undefined;
             this.clientSequenceNumberForLatencyStatistics = undefined;
-            this.opTimeSittingInInboundQueue = undefined;
-            this.durationSittingInInboundQueue = undefined;
+            this.opProcessingTimes[OpTimings.opTimeSittingInboundQueue] = undefined;
```

Comment:
It's better to allocate fully new object, i.e. this.opProcessingTimes = [], or this.opProcessingTimes.length = 0 (assuming it's Ok to clear all thing).

Reply:
I believe it is.